sites/docs/src/content/docs/checklists/community_governance/core_team.mdx

```diff
@@ -42,6 +42,25 @@ Instructions for uploading full-test AWS files for a pipeline to a S3 bucket:
 2.  [ ] Check that large files cannot be reduced or subsetted.
 3.  [ ] Select **Amazon S3-managed keys (SSE-S3)** and create the pipeline directory in the S3 bucket.
 
+## Repository transfer to nf-core organisation
+
+1. [ ] Request ownership of the repo to be transferred from the dev
+    1. If under a organisation repo: ask dev to add the core team member as a co-owner
+    2. If under a user repo: ask dev to directly transfer ownship to core team member's private account
+2. [ ] Transfer the repository to the nf-core organisation
+    1. Go to repository settings
+    2. Bottom of General page
+    3. Press 'Transfer' next to 'Transfer ownership'
+    4. Keep repository name as is (assuming it's correct based on approved name on proposal)
```

Comment:
I would go with if the name is correct keep it, otherwise correct it

Reply:
Done!